examples/src/main/java/org/apache/spark/examples/ml/JavaPipelineExample.java

```diff
@@ -1,88 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.spark.examples.ml;
-
-// $example on$
-import java.util.Arrays;
-
-import org.apache.spark.ml.Pipeline;
-import org.apache.spark.ml.PipelineModel;
-import org.apache.spark.ml.PipelineStage;
-import org.apache.spark.ml.classification.LogisticRegression;
-import org.apache.spark.ml.feature.HashingTF;
-import org.apache.spark.ml.feature.Tokenizer;
-import org.apache.spark.sql.Dataset;
-import org.apache.spark.sql.Row;
-// $example off$
-import org.apache.spark.sql.SparkSession;
-
-/**
- * Java example for simple text document 'Pipeline'.
- */
-public class JavaPipelineExample {
```

Comment:
Yeah, I'll update that in the description too.  Each "Pipeline Example"  is duplicated by "Simple Text Classification Pipeline" in Scala, Python and Java.  The test data is slightly different, but "Pipeline Example" gives false negatives while "Simple Text.." does not.